packages/mds-logger/index.ts

```diff
@@ -217,6 +219,15 @@ async function error(...msg: any[]): Promise<any[]> {
   return censoredMsg
 }
 
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
It may be possible to use `unknown[]` instead of `any[]` to eliminate the need to suppress the `no-explicit-any`